.env

```diff
@@ -0,0 +1,9 @@
+# PostgreSQL Configuration
+DB_USER=your_username
```

Comment:
You pushed .env 🫨, this file should be in .gitignore file, research it and have it only locally, I can steal your passwords and credentials easy.